src/Home/Home.tsx

```diff
@@ -1,5 +1,93 @@
 import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import glamorous from 'glamorous';
+// const Home = () => <h1>Home Page</h1>;
+const { Div } = glamorous;
 
-const Home = () => <h1>Home Page</h1>;
+const CircleButton = glamorous.button({
+  borderRadius: '50%',
+  backgroundColor: 'black',
+  width: '4vw',
+  height: '4vw',
+  padding: 3,
+});
+
+const RectangleThing = glamorous.div({
+  width: 380,
+  height: 250,
+  backgroundColor: 'rgba(255, 255, 255, 0.8)',
+});
+
+// const LinkGrid = glamorous.div({
+//   display:"grid",
+//   grid-template-columns: 400 400 400,
+//   grid-column-gap: 40px,
+
+// })
+class Home extends React.Component {
+  render() {
+    return (
+      <Div margin="5px auto" padding="20px" width="100%">
+        <Div id="nav" display="grid" grid-template-columns="80vw 20vw">
+          <h1> panel.uclaradio.com: links </h1>
+          <Div
+            className="buttonGroup"
+            display="grid"
+            grid-template-columns="5vw 5vw 5vw"
+          >
+            <CircleButton />
+            <CircleButton />
+            <CircleButton />
+          </Div>
+        </Div>
+
+        <Div id="search" margin-bottom="25px">
+          <Div textAlign="center">
+            <input type="text" id="searchBar" color="rgba(255,255,255,.5)" />
+          </Div>
+        </Div>
+
+        <Div
+          className="link-grid"
+          display="grid"
+          grid-template-columns="repeat(3,400px)"
```

Comment:
Let's make CSS grid responsive! Check this out for details: https://medium.com/samsung-internet-dev/common-responsive-layouts-with-css-grid-and-some-without-245a862f48df

Reply:
Do you think inline styles like this are ok? I know we're using glamorous, but I've thinking of starting a css file for Home because I felt that was cleaner. What do you think?

Reply:
or is that defeating the purpose of using glamorous